orcoursetrion/actions/github.py

```diff
@@ -40,7 +45,7 @@ def create_export_repo(course, term, description=None):
     return repo
 
 
-def create_xml_repo(course, term, team, description=None):
+def create_xml_repo(course, term, team, members=None, description=None):
```

Comment:
Adding 'members' complicates this a bit, give it's an api. You could keep create_repo and add/replace_members separate ?  The client (e.g. doof) may want to write a shortcut like this.